pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -200,3 +201,33 @@ def __init__(self, set_variable, item) -> None:
         if set_variable.class_type.element_type != item.class_type:
             raise TypeError("Expecting an argument of the same type as the elements of the set")
         super().__init__(set_variable, item)
+
+#==============================================================================
+class SetUpdate(SetMethod):
+    """
+    Represents a call to the .update() method.
+
+    Represents a call to the .update() method of an object with a set type,
+    Which adds items from another set (or any other iterable).
+    This method is handled through the call to `_visit_SetUpdate` in
```

Comment:
```suggestion
    This method is handled through the call to `_build_SetUpdate` in
```

Does this also need updating for `ListExtend`? Would you mind checking

Reply:
Fixed : 7b7f015

Reply:
Thanks :)